integration/heapster_api_test.go

```diff
@@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
 	return &timeseriesSchema, nil
 }
 
+func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
+	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
```

Comment:
I would rather check the other endpoints in a more mocked and unit-like test for `api/v1/model_handler.go` where I can verify if the returned values are correct.